### PR TITLE
Add contributors page

### LINKED ISSUE #112: Contributors
Add a page that list out all contributors / maintainers

### PR DESCRIPTION
**Pull request by laboon**
closes #112

@wpank @joepetrowski please feel free to issue a PR to add yourselves here.  I didn't have bios for you.

### CODE REVIEW

File: docs/contributors.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+## Anson Lau
+
+Anson is a technical educator at Web3 Foundation. Before that, he worked as a Hyperledger blockchain
+developer to deliver B2B application at IBM. He is passionate about staking, governance,
+cryptoeconomics, and privacy areas. Anson has been actively participating in the blockchain space
+since 2017 and creating video content to help different people to understand how decentralization
+could deliver us a better future.
+
+### Bill Laboon
```
Comment (krichard410) [inline docs/contributors.md]:
```suggestion
## Bill Laboon
```
=)

File: docs/contributors.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+Logan is a Technical Educator at Web3 Foundation where he works on creating material for supporting
+the development of the next generation of distributed technologies. At Web3 Technical Education,
+Logan focuses on creating the Polkadot Wiki as the central source of truth and knowledge for the
+Polkadot protocol. He contributes to the creation of written content and example applications that
+demonstrate how to use these new technologies.
+
+## Kirsten Richard
+
+Kirsten is a Technical Educator intern at the Web3 Foundation from Canada. She's passionate about
```
Comment (laboon) [inline docs/contributors.md]:
```suggestion
Kirsten is a Technical Educator intern at Web3 Foundation from Canada. She's passionate about
```

File: docs/contributors.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
+---
+id: contributors
+title: Contributors
+sidebar_label: Contributors
+---
+
+## Anson Lau
+
+Anson is a technical educator at Web3 Foundation. Before that, he worked as a Hyperledger blockchain
```
Comment (laboon) [inline docs/contributors.md]:
```suggestion
Anson is a Technical Educator at Web3 Foundation. Before that, he worked as a Hyperledger blockchain
```

File: docs/contributors.md (hunk 2 above)
Comment (joepetrowski) [reply docs/contributors.md]: Why not https://github.com/w3f/polkadot-wiki/pull/878/files#diff-9558ae512b5cea68efed66b2b4b0b233R17 ?

Comment (laboon) [reply docs/contributors.md]: Because I'm a hypocrite, apparently.

File: docs/contributors.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+Anson is a Technical Educator at Web3 Foundation. Before that, he worked as a Hyperledger blockchain
+developer to deliver B2B application at IBM. He is passionate about staking, governance,
+cryptoeconomics, and privacy areas. Anson has been actively participating in the blockchain space
+since 2017 and creating video content to help different people to understand how decentralization
+could deliver us a better future.
+
+## Bill Laboon
+
+Bill Laboon is the Technical Education Lead at the Web3 Foundation, after spending the previous five
```
Comment (laboon) [inline docs/contributors.md]:
```suggestion
Bill Laboon is the Technical Education Lead at Web3 Foundation, after spending the previous five
```

File: docs/contributors.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+Friendly Introduction to Software Testing, an undergraduate textbook; and Strength in Numbers, a
+near-future novel set in a world in which cryptocurrency has eliminated traditional money.
+
+## Bruno Skvorc
+
+Bruno is a blockchain developer and educator from Croatia. He joined the ecosystem full time in 2015
+with the advent of Ethereum, and created educational resources, programming courses, tutorials and
+newsletters for budding developers of the industry. Since 2018 he had been working on one of the
+teams building the next generation of Ethereum, and since 2019 he's been a technical educator at
+Web3 Foundation, dedicated to forwarding the web3 vision and helping make sure everyone, everywhere
```
Comment (krichard410) [inline docs/contributors.md]:
```suggestion
Web3 Foundation, dedicated to forwarding the Web3 vision and helping make sure everyone, everywhere
```

**Review by krichard410 — APPROVED** [review]
This looks great to me. Did anyone else want to add in a bio?

**Comment by laboon** [comment]
They can always add it later as a separate PR.

**Pull request merged**